src/main/js/wiki/components/FileBrowser.tsx

```diff
@@ -32,7 +32,7 @@ export const orderFiles = (files) => {
   });
 };
 
-class FileBrowser extends React.Component<Props> {
+class FileBrowser extends React.Component<Props, any> {
```

Comment:
why adding any here?

Reply:
TBH if have no clue how it got here in the first place. I am going to remove it.

Reply:
- [x] done